src/CitizenProject/CitizenProjectManager.php

```diff
@@ -499,50 +498,49 @@ public function hasCitizenProjectInStatus(Adherent $adherent, array $status): bo
     }
 
     /**
-     * Uploads and saves the citizen project image to the cloud storage.
+     * Uploads and saves the citizen project image.
      *
      * @param CitizenProject $citizenProject
      */
     public function addImage(CitizenProject $citizenProject): void
     {
-        if ($citizenProject->getImage() instanceof UploadedFile) {
-            $path = $citizenProject->getImagePath();
-
-            // Clears the old image if needed
-            if (null !== $citizenProject->getImageName() && $oldImagePath = $citizenProject->getImagePath()) {
-                $this->storage->delete($oldImagePath);
-            }
+        // Clears the old image if needed
+        if (null !== $citizenProject->getImageName() && $oldImagePath = $citizenProject->getImagePath()) {
+            $this->storage->delete($oldImagePath);
+        }
 
-            $citizenProject->setImageName($citizenProject->getImage());
+        $citizenProject->setImageName($citizenProject->getImage());
```

Comment:
You removed the check that ensure that `$citizenProject->getImage()` is an instance of `UploadedFile`, is this intended?